src/control/lib/control/check.go

```diff
@@ -414,9 +413,6 @@ func (r *SystemCheckReport) RepairChoices() []*SystemCheckRepairChoice {
 		}
 	}
 
-	sort.Slice(choices, func(i, j int) bool {
-		return choices[i].Action < choices[j].Action
-	})
```

Comment:
Won't this result in the human-readable choices being kind of jumbled too?

Reply:
The recommended action will be the first one. Does the order matter otherwise?